pkg/action/list.go

```diff
@@ -154,7 +156,7 @@ func (l *List) Run() ([]*release.Release, error) {
 			return nil, err
 		}
 	}
-
+	os.Setenv("helm-serviceName", l.ServiceName)
```

Comment:
Setting an environment variable to be picked up later is a bit of a hack. We don't pass information that way in Helm. It would need to be handled through passing through function signatures.

Reply:
Yes, this is a hack and it's not meant to merge. I was hoping for any suggestion on implementing what I want to do in a right way. As of now, there's no way to change the behavior of driver's output.